axolotl-web/src/pages/Register.vue

```diff
@@ -51,19 +51,37 @@ import WarningMessage from "@/components/WarningMessage";
 import { mapState } from "vuex";
 
 export default {
-  name: "register",
+  name: "RegisterPage",
   components: {
     VueTelInput,
     WarningMessage,
   },
-  props: {
-    msg: String,
+  data() {
+    return {
+      phone: "",
+      infoPage: true,
+    };
+  },
+  computed: mapState([
+    "gui",
+    "ratelimitError",
+    "registrationStatus",
+    "captchaToken",
+    "captchaTokenSent",
+  ]),
+  mounted() {
+    var userLang = navigator.language || navigator.userLanguage;
+    this.$language.current = userLang;
+    if (this.captchaToken != null && !this.captchaTokenSent) {
```

Comment:
Instead of using the fuzzy `!=` check, I suggest to instead use the strict comparison (`!==`) here.